CHANGELOG.md

```diff
@@ -63,6 +63,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Sets image build status for Milano compatibility ([#472][] @viniciusgama)
 - Add node binding for node-sass ([#465][] @thestephenmarshall)
 - Enable ESLint in overcommit ([#462][] @thestephenmarshall)
+- Display wrong numbers in Person Contact Kit ([#499][] @KFad11)
```

Comment:
Hey @KFad11 looks great, this should be moved to the `Unreleased` section though, under the `### Fixed` category.

Reply:
Ah HAH! I get it now. Should be good to go.